pallets/security/src/lib.rs

```diff
@@ -308,6 +308,9 @@ impl<T: Config> Pallet<T> {
 	}
 
 	/// for testing purposes only!
+	/// Std is never enabled in the Runtime.
+	/// This function can only be called from testing but not from the pallet itself:
+	#[cfg(feature = "std")]
```

Comment:
Can we use `#[cfg(test)]` instead or does this not work for some reason?

Reply:
I tried. No. We can't.

Reply:
It does not work because when you use `#[cfg(test)]` this code will not include to binary and other packages can't reach this function.

Reply:
you can restrict it to testing, which only works from the **same crate** since the
'test' feature **does not propagate** to dependencies:
```
#[cfg(test)]
pub fn some_fn(n: T::BlockNumber) {
    SomeStorage::<T>::set(n);
}
```

but other packages use this function. so we can't use `#[cfg(test)]` in current case
@ebma

Reply:
Okay got it. But can we use the 'testing-utils' feature for this then, like we do in other crates, eg stellar-relay? In stellar-relay some helper functions that are also used in other crates [are exposed](https://github.com/pendulum-chain/spacewalk/blob/debc953b78986c011ca55b265c808e97db61655b/pallets/stellar-relay/src/lib.rs#L19-L20) when the 'testing-utils' feature is imported, maybe we can do something similar here. 

I am sure your approach with `std` also works, but it seems a bit hacky and I think we can do better by using another feature.

Reply:
Wow! Amazing! You are right. 
Thank you so much. I changed it. 
use `#[cfg(feature = "testing-utils")]` instead of `#[cfg(feature = "std")]`
Ready for merge.
@ebma